Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -860,12 +616,15 @@ - (void)scrollToUnreadIndicatorAnimated
     [self updateLastVisibleTimestamp];
 }
 
+// TODO: We need to audit every usage of this method.
 - (void)resetContentAndLayout
```

Comment:
I'm pretty sure we can remove much usage of this method without sacrificing correctness, which will be a real perf win.  TODO.